main.cpp

```diff
@@ -246,6 +247,10 @@ void drawcredits()
     // Make the backbuffer visible
 }
 
+int autosizer() {
```

Comment:
a more identifiable name would be better here. e.g. `getAutoSegmentSize`

Reply:
I would also put this in an anonymous namespace in `Config.cpp` since it's not used anywhere else. Not only does this avoid cluttering another TU, but it also allows the compiler to inline it when it's used (which can't be done with functions that are defined in a different TU).